[2024-05-05T15:59:30.341+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T15:59:30.831+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-10T00:00:00+00:00 [queued]>
[2024-05-05T15:59:30.945+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-10T00:00:00+00:00 [queued]>
[2024-05-05T15:59:30.972+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T15:59:31.242+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-10 00:00:00+00:00
[2024-05-05T15:59:31.452+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=529) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-05T15:59:31.482+0000] {standard_task_runner.py:63} INFO - Started process 575 to run task
[2024-05-05T15:59:31.458+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'scheduled__2024-04-10T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp5616w9_7']
[2024-05-05T15:59:31.514+0000] {standard_task_runner.py:91} INFO - Job 49: Subtask get_files
[2024-05-05T15:59:32.012+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.get_files scheduled__2024-04-10T00:00:00+00:00 [running]> on host edcd6c8e3dad
[2024-05-05T15:59:33.111+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-10T00:00:00+00:00'
[2024-05-05T15:59:33.158+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T15:59:33.162+0000] {logging_mixin.py:188} INFO - 0 files found in /opt/***/dags/data
[2024-05-05T15:59:33.166+0000] {python.py:237} INFO - Done. Returned value was: []
[2024-05-05T15:59:33.169+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T15:59:33.346+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240410T000000, start_date=20240505T155930, end_date=20240505T155933
[2024-05-05T15:59:33.755+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T15:59:34.016+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05T15:59:34.031+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
